File: src/main/java/com/example/inventoryAppSpring/models/Product.java
package com.example.inventoryAppSpring.models;

import lombok.*;
import javax.persistence.*;

@Data
@AllArgsConstructor @NoArgsConstructor @Getter
@Setter
@Entity @Table(name = "PRODUCTS")
public class Product {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "PRODUCT_NAME")
    private String productName;

    @Column(name = "PRODUCT_QUANTITy")
    private int productQuantity;
}

File: client/src/components/Product.js
import React from 'react';

const Product = (props) => {
	console.log('product props', props)
	return (
		<div>
			<h4>
			  {props.product.productName}:  
			  <small>{props.product.productQuantity}</small>
		    </h4>
		</div>
	)
}

export default Product;